src/js/components/devices/device-details/devicesystem.js

```diff
@@ -71,6 +73,7 @@ export const DeviceSystem = ({
         <TwoColumnData config={{ 'Server IP': deviceIp }} compact setSnackbar={setSnackbar} />
       </DeviceDataCollapse>
       <DeviceDataCollapse className={classes.container} title="System for this gateway">
+        <EnterpriseNotification isEnterprise={hasFullFiltering} benefit="see devices connected to your gateway device for easy access" />
```

Comment:
Didn't get this one: what is it?

Reply:
Once we have reporting, all plans will be able to use all filtering options, until then at least users on the basic plan won't be able to see devices from systems (since retrieving them relies on `$ne`) - unlikely to be shown often, but was part of the reverted commit...

Reply:
FYI: systems are Enterprise only as they rely on Mender Gateway.

Reply:
Right, I had the feeling in the back of my head, but was misled by the code, since the rest just relied on less explicit checks for this => https://github.com/mendersoftware/gui/pull/3664